### PR TITLE
Updating Text docs

### LINKED ISSUE #36: Update text apis docs
Docs for the Text APIs is outdated, we must update it to the latest implementation

### PR DESCRIPTION
**Pull request by erickzanardo**
Fixes #36

### CODE REVIEW

File: doc/text.md [hunk 1]
```diff
@@ -0,0 +1,90 @@
...
+    Color color = material.Colors.black;
+```
+
+After the creation of the config you can use its `render` method to draw some string on a canvas:
+
+```dart
+config.render(canvas, Offset(size.width - 10, size.height - 10);
+```
+
+## Text Components
```
Comment (luanpotter) [inline doc/text.md]: I would change the order, as TextComponent is simpler and easier to grasp before TextBoxComponent.

File: doc/text.md [hunk 2]
```diff
@@ -0,0 +1,90 @@
...
+
+You can use Dart's hexadecimal notation to make it really easy; for instance: `0xFF00FF00` is fully opaque green (the 'mask' would be `0xAARRGGBB`).
+
+There is a color enum to make it easy to use common colors; it is in the material flutter package:
+
+```dart
+    import 'package:flutter/material.dart' as material;
+
+    Color color = material.Colors.black;
+```
```
Comment (luanpotter) [inline doc/text.md]: I like this explanation, but we could take the opportunity to link to a Color Palette documentation to instruct to better using colors and paints :)

Comment (erickzanardo) [reply doc/text.md]: I am not very aware of the palette APIs, will merge this right now and we do the palette docs later. This way at least we remove the old and removed part of the text doc :D

**Comment by erickzanardo** [comment]
Updating text documentation

**Pull request merged**